packages/connector-jsdom/tests/get-element-final-url.ts

```diff
@@ -0,0 +1,100 @@
+import * as url from 'url';
+
+import test from 'ava';
+import { EventEmitter2 } from 'eventemitter2';
+
+import { test as testUtils } from '@hint/utils';
+import { Server } from '@hint/utils-create-server';
+
+import { Engine, IConnector } from 'hint';
+
+import JSDOMConnector from '../src/connector';
+
+const { generateHTMLPage } = testUtils;
+
+test(`[jsdom] getElementFinalUrl returns the same element URL if the URL is absolute`, async (t) => {
+    const baseUrl = 'http://baseurl.com/';
+    const elementUrl = 'http://absolute.com/test.png';
+    const engine = new EventEmitter2({
+        delimiter: '::',
+        maxListeners: 0,
+        wildcard: true
+    }) as Engine;
+
+    const connector: IConnector = new JSDOMConnector(engine, {});
+    const server = await Server.create({ configuration: generateHTMLPage(`<base href="${baseUrl}">`, '') });
+
+    await connector.collect(new URL(`http://localhost:${server.port}/`));
+
+    await engine.emitAsync('element::base', {
```

Comment:
This should be able to be removed from the test since we now get the base URL from the DOM snapshot instead of during traversal.

Reply:
yes, I just forgot to remove them from some tests.